lib/core/form/services/activiti-alfresco.service.ts

```diff
@@ -55,14 +55,26 @@ export class ActivitiContentService {
     /**
      * Returns a list of all the repositories configured
      *
-     * @param accountId
-     * @param folderId
      */
-    getAlfrescoRepositories(tenantId: number, includeAccount: boolean): Observable<any> {
```

Comment:
will be better leave the old method with the option, this is a breaking change and honestly have two method just to change a parameter I don't like is better have the possibility to pass it has option includeAccounts